vars/fcosBuild.groovy

```diff
@@ -33,14 +33,14 @@ def call(params = [:]) {
             }
         }
 
-        def extraFetchArgs = params.get('extraFetchArgs', "");
-        shwrap("cd ${cosaDir} && cosa fetch --strict ${extraFetchArgs}")
+        def extraFetchArgs = params.get('extraFetchArgs', "--strict");
```

Comment:
I think normally you do always want strict mode turned on in CI because it ensures that the lockfiles are the complete sources of truth (see https://github.com/coreos/rpm-ostree/pull/1858 and https://github.com/coreos/fedora-coreos-tracker/issues/454). Essentially, non-strict mode is useful mostly for the developer case.

In the case of RHCOS, which doesn't use lockfiles yet, using `cosa {fetch,build} --strict` should have no effect. 

That said, it does make sense to make this configurable to be flexible (and e.g. allow developers to hack up the manifest quickly and fire up a Jenkins job for it).

Instead of putting this in `extraFetchArgs` and `extraArgs`, and potentially having callers which already use these lose the `--strict` arg, WDYT about adding a new `strict` bool instead which defaults to `true`?

Reply:
Thanks for the overview @jlebon ;)